ODIN_II/SRC/odin_ii.cpp

```diff
@@ -278,34 +280,48 @@ netlist_t *start_odin_ii(int argc,char **argv)
 		}
 	}
 
-	/*************************************************************
-	 * begin simulation section
-	 */
 	netlist_t *odin_netlist = NULL;
 
-	if(global_args.blif_file.provenance() == argparse::Provenance::SPECIFIED
-	|| global_args.interactive_simulation 
-	|| global_args.sim_num_test_vectors 
-	|| global_args.sim_vector_input_file.provenance() == argparse::Provenance::SPECIFIED)
+	/* read the blif */
+	if(global_args.blif_file.provenance() == argparse::Provenance::SPECIFIED || global_args.sim_vector_input_file.provenance() == argparse::Provenance::SPECIFIED)
 	{
 		// if we started with a verilog file read the output that was made since
 		// the simulator can only simulate blifs
-		if(global_args.blif_file.provenance() != argparse::Provenance::SPECIFIED)
-		{
-			configuration.list_of_file_names = { global_args.output_file };
-			current_parse_file =0;
-		}
-
-		try 
```

Comment:
this should be kept and maybe moved above this block, but nonetheless it is needed unless you change the read blif function.